webview/src/views/main.svelte

```diff
@@ -9,7 +9,6 @@
     }
     const data = Object.fromEntries(new FormData(e.target))
 
-    // TODO: implement this
     vscode.postMessage({
       type: "startFromId",
       ...data,
```

Comment:
I have been unable to debug from a history file locally.  I'm getting an `Unexpected token in JSON` error and I'm wondering if it's an issue with my file or the way we are handling the buffer that is eventually passed to `historyFromJSON` 🤔

Reply:
Let's look into this when we meet.